composer/datasets/utils.py

```diff
@@ -32,8 +34,8 @@ class NormalizationFn:
     Args:
         mean (Tuple[float, float, float]): The mean pixel value for each channel (RGB) for the dataset.
         std (Tuple[float, float, float]): The standard deviation pixel value for each channel (RGB) for the dataset.
-        ignore_background (bool): If ``True``, ignore the background class in the training loss.
-            Only used in semantic segmentation. Default is ``False``.
+        ignore_background (bool): If ``True``, ignore the background class in the training
+            loss. Only used in semantic segmentation. Default: ``False``.
```

Comment:
Formatting of default for the following line in composer.datasets.utils.add_vision_dataset_transform

> Function to be added to the dataset’s collection of transforms.

Reply:
For pil_image_collate 

Default for memory_format.